### ISSUE TITLE
User Bug Report

### ISSUE THREAD

**Issue by rkrajnc** [rkrajnc/minimig-mist#23]
Submitter: Richard
Email: computergeek43212@gmail.com

```
minimig-mist release: AGA 1.0
firmware version: latest release from github website as of today
kickstart version: 1.3
chipRAM: 512kb
fastRAM: 0
slowRAM: 512kb
game / demo: Rainbow Islands (1990)(Ocean)[cr SCX][t +8 Flashtro]
other system tested: WinUAE - works corrently in WinUAE

description of the problem: When the flashtro menu displays on screen the mouse does not respond correctly and jumps straight to the first and last cheat option. Selecting Off or On takes holding down mouse buttons and dragging the mouse until it may randomly select start game or the first cheat only.
```

This happens to all games that use flashtro trainers, the minimig code has some fix for this but seems to not be working with the current AGA build.

2008-07-28 - JB - USERIO - added JOYTEST register to make it compatible with ALPHA1/SIRIAX flashtro/trainer - No longer working

**Comment by rkrajnc** [comment]
That JOYTEST fix should already be in minimig-mist. I think we have some problems with mice/keyboards and how they are interfaced to the minimig-mist core. I'll look into it, as soon as I finish working on the IDE problems.

**Comment by rkrajnc** [comment]
Actually, it seems the problem was that the JOYTEST values were not applied to USB mouse inputs correctly, I fixed that, will be available in the new core.

**Comment by rkrajnc** [comment]
Fixed in minimig-mist-1_1 (52f1216a899418c61dd1a10840b3f0622af46bc5).
